common/version.go

```diff
@@ -4,23 +4,11 @@ import "github.com/nspcc-dev/neo-go/pkg/interop/native/std"
 
 const (
 	major = 0
-	minor = 16
-	patch = 1
-
-	// Versions from which an update should be performed.
-	// These should be used in a group (so prevMinor can be equal to minor if there are
-	// any migration routines.
```

Comment:
> We can't safely upgrade from 0.14.0

Why? If there is a missing migration code, let's add it. Otherwise I can't test upgrade from version `<v0.16`. If we leave step-by-step update, then test framework should skip contracts `<common.PrevVersion`.

Reply:
> there is a missing migration code

That's the case. But we don't have any pre-0.15.x networks in production and the upgrade procedure up to 0.16.0 was strictly sequential, so we may forget about pre-0.15.x at least. At the same time we still need this check, because we won't keep transition code forever. In a year we'll start dropping these old migrations and that's where we'll be bumping the minimal upgrade version.